invenio_github/models.py

```diff
@@ -220,29 +226,32 @@ def disable(cls, user_id, github_id, name):
         :param repo_id: GitHub id of the repository.
         :param name: Fully qualified name of the repository.
         """
-        repo = cls.get(user_id, github_id=github_id, name=name)
-        if repo:
-            repo.enabled = False
-            repo.user_id = None
+        try:
+            repo = cls.get(user_id, github_id=github_id, name=name)
             repo.hook = None
-            db.session.add(repo)
+            repo.user_id = None
+        except NoResultFound:
```

Comment:
I'm not sure. I think it's an error if the repo does not exists so we should raise.